src/index.html

```diff
@@ -9,7 +10,7 @@
 
 <body>
     <div id="root"></div>
-    <script src="./dist/bundle.js"></script>
+    <script src="../dist/bundle.js"></script>
```

Comment:
After adding HtmlWebpackPlugin- delete this script. Scripts should be added by webpack while bundling.